python_modules/dagster/dagster/_legacy/__init__.py

```diff
@@ -8,22 +8,12 @@
     PresetDefinition as PresetDefinition,
     build_assets_job as build_assets_job,
     default_executors as default_executors,
-    pipeline as pipeline,
 )
 from dagster._core.execution.api import (
-    execute_pipeline as execute_pipeline,
-    execute_pipeline_iterator as execute_pipeline_iterator,
-    reexecute_pipeline as reexecute_pipeline,
+    _reexecute_job as _reexecute_job,
```

Comment:
Does this API exist? The pythonic way to perform re-execution should be ReexecutionOptions on `execute_job` now

Reply:
yeah that ended up there through some fluke, deleted